mcp/k8s_cron_job.tf

```diff
@@ -209,13 +209,12 @@ resource "kubernetes_cron_job" "self" {
               }
             }
             dynamic "container" {
-              for_each = lookup(each.value.cron_job.spec.job_template.spec.template.spec, "container", []) == [] ? [] : [for container in each.value.cron_job.spec.job_template.spec.template.spec.container : {
+              for_each = [for container in lookup(each.value.cron_job.spec.job_template.spec.template.spec, "container", []) : {
```

Comment:
Don't need a condition here any more?

Reply:
This was the line that if the list was configured incorrectly would cause a terraform crash.
The new format used here had been used in the pod block for `kubernetes_deployment`, is still conditional, and avoids the crash if things are formatted incorreclty.